modules/ixBidAdapter.js

```diff
@@ -285,17 +285,19 @@ function getBidRequest(id, impressions) {
  * From the userIdAsEids array, filter for the ones our adserver can use, and modify them
  * for our purposes, e.g. add rtiPartner
  * @param {array} allEids userIdAsEids passed in by prebid
+ * @param {object} flocId flocId passed in by prebid
  * @return {object} contains toSend (eids to send to the adserver) and seenSources (used to filter
  *                  identity info from IX Library)
  */
-function getEidInfo(allEids) {
+function getEidInfo(allEids, flocData) {
   // determines which eids we send and the rtiPartner field in ext
   var sourceRTIMapping = {
```

Comment:
Any reason not to have this be a file const, so that we do not have the cost of instantiating on every call to getEidInfo? Could lazily initialize if this is not something we'd expect to run on most flows.

Also prefer const/let

Reply:
the code has been updated to use const/let in the scope of the pull request.